app/services/moderator/merge_user.rb

```diff
@@ -45,6 +45,8 @@ def update_social
         @delete_user.update_columns(twitter_username: nil, github_username: nil)
         @keep_user.update_columns(twitter_username: @old_tu) if @keep_user.twitter_username.nil?
         @keep_user.update_columns(github_username: @old_gu) if @keep_user.github_username.nil?
+        @keep_user.touch(:profile_updated_at)
```

Comment:
You can touch both with 1 request: ` @keep_user.touch(:profile_updated_at, :last_followed_at)`